txpipe/data_types/types.py

```diff
@@ -54,19 +75,8 @@ class RandomsCatalog(HDFFile):
 
```

Comment:
does the random files have to have e1/e2? seems strange to require e.g. randoms for clustering to have e1/e2

Reply:
No, it doesn't, good point.  If we do need shear randoms later this won't work because we need to use a different tomography catalog.

I've pushed a change to fix this.